cmd/buck/cli/archive.go

```diff
@@ -20,6 +20,14 @@ var bucketArchiveCmd = &cobra.Command{
 		}
 	},
 	Run: func(c *cobra.Command, args []string) {
+		cmd.Warn("Warning! Archiving is based on an experimental testnet. Your data may be lost at any time.")
+		prompt := promptui.Prompt{
+			Label:     "Proceed",
+			IsConfirm: true,
+		}
+		if _, err := prompt.Run(); err != nil {
+			cmd.End("")
+		}
```

Comment:
Some extra warning. Leveraged suggestion by @sanderpick

Reply:
Possibly more explicit: _Warning! Archives are currently saved on an experimental test network. They may be lost at any time._

Reply:
Changing.